styles.css

```diff
@@ -0,0 +1,689 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300;400;700;900&family=Roboto:wght@100;300;400;500;700;900&display=swap');
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media queries
+ /*!///////////////////////!*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: "Merriweather", serif;
+}
+
+body,
+button,
+input,
+textarea {
+    font-family: "Roboto", sans-serif;
+}
+
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --standard-text: #0C0C0C;
+    --floating-words: #F2F0FF;
+    --project-background: #F3F6F9;
+    --service-tile: #7A7A7A;
+}
+
+body {
+    font-size: 16px;
+    min-height: 100vh;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+h1 {
+    font-size: 2.25rem;
+    font-weight: 700;
+}
+
+h2,
+h3 {
+    font-size: 1.5rem;
+}
+
+header p {
```

Comment:
Als dingen alleen gelden voor de tekst in de header of alleen voor de tekst in het contact blok, is het geen algemene typografie meer. Je wil dus een 'standaard' kiezen voor alle tekst (bijvoorbeeld `1.25`) en die hier generiek declareren - en vervolgens alles dat afwijkt overschrijven met een specifieke class, zoals `.header-text` o.i.d.